pkgs/games/re3/default.nix

```
@@ -0,0 +1,42 @@
+{ branch ? "re3"
+, callPackage
+, fetchFromGitHub
+}:
+
+{
+  re3 = callPackage ./base.nix rec {
+    pname = "re3";
+    version = "12-07-2021";
```

```suggestion
    version = "unstable-12-07-2021";
```
Why are we not using the same source for all packages?